components/Canvas/FilterData/index.js

```diff
@@ -0,0 +1,23 @@
+import { MenuItem, Select } from '@mui/material'
+import React from 'react'
+
+export default function Filter({setFilterValue}) {
+
+  function handleFilter(event) {
+    setFilterValue(event.target.value)
+    console.log(event.currentTarget)
+  }
+
+  return (
+      <Select
+        id="demo-simple-select"
+        label="Age"
```

Comment:
Better write lowercase?

```suggestion
        label="age"
```